### PR TITLE
Bump gl-native 10.8.0-beta.2

### PR DESCRIPTION
**Pull request by Chaoba**
## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `make update-api` to update generated api files, if there's public API changes, otherwise the `verify-api-*` CI steps might fail.
 - [x] Update [CHANGELOG.md](../CHANGELOG.md) or use the label 'skip changelog', otherwise `check changelog` CI step will fail.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,9 +17,11 @@ Mapbox welcomes participation and contributions from everyone.
 * Fix Android memory leak when destroying platform view annotation manager. ([1568](https://github.com/mapbox/mapbox-maps-android/pull/1568))
 * Fix style getters for terrain, light and atmosphere resetting properties. ([1573](https://github.com/mapbox/mapbox-maps-android/pull/1573))
 * Fix possible ANR when destroying renderer. ([1567](https://github.com/mapbox/mapbox-maps-android/pull/1567))
+* Fix elevation of pole geometry when exaggerated terrain is used. ([1574](https://github.com/mapbox/mapbox-maps-android/pull/1574))
 
 ## Dependencies
 Bump gl-native to v10.8.0-beta.1, common to v23.0.0-beta.1. ([1568](https://github.com/mapbox/mapbox-maps-android/pull/1568))
+Bump gl-native to v10.8.0-beta.2. ([1574](https://github.com/mapbox/mapbox-maps-android/pull/1574))
```
Comment (kiryldz) [inline CHANGELOG.md:24]: Guess we should actually replace entry above and note that 10.8.0-beta.2 is used

Comment (Chaoba) [reply CHANGELOG.md:24]: Fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,9 +17,11 @@ Mapbox welcomes participation and contributions from everyone.
 * Fix Android memory leak when destroying platform view annotation manager. ([1568](https://github.com/mapbox/mapbox-maps-android/pull/1568))
 * Fix style getters for terrain, light and atmosphere resetting properties. ([1573](https://github.com/mapbox/mapbox-maps-android/pull/1573))
 * Fix possible ANR when destroying renderer. ([1567](https://github.com/mapbox/mapbox-maps-android/pull/1567))
+* Fix elevation of pole geometry when exaggerated terrain is used. ([1574](https://github.com/mapbox/mapbox-maps-android/pull/1574))
 
 ## Dependencies
-Bump gl-native to v10.8.0-beta.1, common to v23.0.0-beta.1. ([1568](https://github.com/mapbox/mapbox-maps-android/pull/1568))
+Bump common to v23.0.0-beta.1. ([1568](https://github.com/mapbox/mapbox-maps-android/pull/1568))
+Bump gl-native to v10.8.0-beta.2. ([1574](https://github.com/mapbox/mapbox-maps-android/pull/1574))
```
Comment (kiryldz) [inline CHANGELOG.md:24]:
```suggestion
Bump gl-native to v10.8.0-beta.2 and common to v23.0.0-beta.1. ([1574](https://github.com/mapbox/mapbox-maps-android/pull/1574), [1568](https://github.com/mapbox/mapbox-maps-android/pull/1568))
```

**Review by yunikkk — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,9 +17,10 @@ Mapbox welcomes participation and contributions from everyone.
 * Fix Android memory leak when destroying platform view annotation manager. ([1568](https://github.com/mapbox/mapbox-maps-android/pull/1568))
 * Fix style getters for terrain, light and atmosphere resetting properties. ([1573](https://github.com/mapbox/mapbox-maps-android/pull/1573))
 * Fix possible ANR when destroying renderer. ([1567](https://github.com/mapbox/mapbox-maps-android/pull/1567))
+* Fix elevation of pole geometry when exaggerated terrain is used. ([1574](https://github.com/mapbox/mapbox-maps-android/pull/1574))
 
 ## Dependencies
-Bump gl-native to v10.8.0-beta.1, common to v23.0.0-beta.1. ([1568](https://github.com/mapbox/mapbox-maps-android/pull/1568))
+Bump gl-native to v10.8.0-beta.2 and common to v23.0.0-beta.1. ([1574](https://github.com/mapbox/mapbox-maps-android/pull/1574), [1568](https://github.com/mapbox/mapbox-maps-android/pull/1568))
```
Comment (yunikkk) [inline CHANGELOG.md:23]: Imo we can omit 1568, since we don't want to share gl-native first beta in any way

Comment (Chaoba) [reply CHANGELOG.md:23]: But common sdk was updated in 1568

**Review by pengdev — APPROVED** [review]

**Review by kiryldz — APPROVED** [review]

**Pull request merged**